examples/demo.js

```diff
@@ -211,6 +214,18 @@ window.onload = function () {
     download(image, 'index.gif', 'image/gif');// downloadjs library function
   }
 
+  function savetopubliclab(){
```

Comment:
See above: The name should be camel case. There should also be a space between the () and {